### ISSUE TITLE
3.4.4 / 3.4.5 Bug sorting custom navigation

### ISSUE THREAD

**Issue by tim-bec** [contao/core#7785]
we have noticed a bug in the custom navigation modul in the sorting / unpack

# How can you reproduce this in the online demo?

* Login
* Make sure error message are displayed
* Create a custom navigation module
* Select some pages
* save module
* show details of module

# How does the error effect the user?

In our case this error seems to result in an internal server error / fatal out of memory in the frontend. 

# Error Backend
( Take a look at the sorting: , , , )

![screenshot 2015-04-29 01 05 47](https://cloud.githubusercontent.com/assets/1793396/7382195/386696b2-ee0c-11e4-8e33-5a97cf5ec982.png)

# Stacktrace
```
Warning: unpack() [function.unpack]: Type H: not enough input, need 4, have 2 in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'unpack() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: implode() [function.implode]: Invalid arguments passed in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'implode() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: unpack() [function.unpack]: Type H: not enough input, need 4, have 2 in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'unpack() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: implode() [function.implode]: Invalid arguments passed in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'implode() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: unpack() [function.unpack]: Type H: not enough input, need 4, have 1 in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'unpack() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: implode() [function.implode]: Invalid arguments passed in system/modules/core/library/Contao/String.php on line 557
#0 [internal function]: __error(2, 'implode() [show()
#4 system/modules/core/controllers/BackendMain.php(126): Contao\Backend->getBackendModule('themes')
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: Cannot modify header information - headers already sent by (output started at /home/www/demo/system/helper/functions.php:68) in system/modules/core/library/Contao/Template.php on line 255
#0 [internal function]: __error(2, 'Cannot modify h...', '...', 255, Array)
#1 system/modules/core/library/Contao/Template.php(255): header('Vary: User-Agen...', false)
#2 system/modules/core/classes/BackendTemplate.php(113): Contao\Template->output()
#3 system/modules/core/controllers/BackendMain.php(268): Contao\BackendTemplate->output()
#4 system/modules/core/controllers/BackendMain.php(130): Contao\BackendMain->output()
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}

Warning: Cannot modify header information - headers already sent by (output started at /home/www/demo/system/helper/functions.php:68) in system/modules/core/library/Contao/Template.php on line 256
#0 [internal function]: __error(2, 'Cannot modify h...', '...', 256, Array)
#1 system/modules/core/library/Contao/Template.php(256): header('Content-Type: t...')
#2 system/modules/core/classes/BackendTemplate.php(113): Contao\Template->output()
#3 system/modules/core/controllers/BackendMain.php(268): Contao\BackendTemplate->output()
#4 system/modules/core/controllers/BackendMain.php(130): Contao\BackendMain->output()
#5 contao/main.php(20): Contao\BackendMain->run()
#6 {main}
```

**Comment by leofeyer** [comment]
Fixed in 718910d82c98663e51d1b92c94fbe48c6522ecd1.

**Comment by tim-bec** [comment]
thanks for the fast fix! does this fix also occur in 3.4.x ?

**Comment by leofeyer** [comment]
There won't be another version 3.4 release. Contao 3.5 is coming next month.
